sdk/core/azure-core-test/CMakeLists.txt

```diff
@@ -10,28 +10,25 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 set(
   AZURE_CORE_TEST_HEADER
-  inc/azure/core/test/interceptor_manager.hpp
   inc/azure/core/test/network_models.hpp
-  inc/azure/core/test/playback_http_client.hpp
-  inc/azure/core/test/record_network_call_policy.hpp
   inc/azure/core/test/test_base.hpp
   inc/azure/core/test/test_context_manager.hpp
+  inc/azure/core/test/test_proxy_manager.hpp
 )
 
 set(
   AZURE_CORE_TEST_SOURCE
   src/private/package_version.hpp
-  src/interceptor_manager.cpp
-  src/playback_http_transport.cpp
-  src/record_policy.cpp
+  src/test_proxy_policy.cpp
   src/test_base.cpp
+  src/test_proxy_manager.cpp
 )
 
 add_library (
   azure-core-test-fw
   ${AZURE_CORE_TEST_HEADER}
   ${AZURE_CORE_TEST_SOURCE}
-  )
+   "src/test_proxy_manager.cpp" "src/test_proxy_policy.cpp")
```

Comment:
This is unnessecary.  These two files are part of AZURE_CORE_TEST_SOURCE

Reply:
auto added by VS